Scripts/datahandling/zone_variables.json

```diff
@@ -33,7 +33,11 @@
         "area_university_buildings",
         "students_comprehensive",
         "students_upper_secondary",
-        "students_higher_education"
+        "students_higher_education",
+        "gen_marita",
+        "gen_kalevi",
+        "att_marita",
+        "att_kalevi"
```

Comment:
Could you add these to data in 08. Hankkeessa tuotetut lähtötiedot / Model-system lähtödata before merging? Otherwise model runs will fail.